helpers/setup_wrapper.sh

```diff
@@ -0,0 +1,5 @@
+#! /bin/bash
+
+set -e
+
+python3 ${OPS_ROOT}/setup-spark-k8s.py "$@"
```

Comment:
this file is not needed
1) we should put `#!/usr/bin/env python3` as shebang for `setup-spark-k8s.py`
2) set 755 rights for `setup-spark-k8s.py`
3) expose `setup-spark-k8s.py` instead of `setup_wrapper.sh`